packages/patternfly-3/patternfly-react/src/components/DualListSelector/DualList.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { defaultTemplate } from 'storybook/decorators/storyTemplates';
+import { storybookPackageName, DOCUMENTATION_URL, STORYBOOK_CATEGORY } from 'storybook/constants/siteConstants';
+import { name } from '../../../package.json';
+import { DualListControlled } from './index';
+import { items, dropdownItems } from './DualListMocks';
+
+const stories = storiesOf(
+  `${storybookPackageName(name)}/${STORYBOOK_CATEGORY.FORMS_AND_CONTROLS}/Dual List Selector`,
+  module
+);
+
+stories.addDecorator(
+  defaultTemplate({
+    title: 'Dual List Selector',
+    documentationLink: `${DOCUMENTATION_URL.PATTERNFLY_ORG_FORMS}dual-list-selector/`
+  })
+);
+
+stories.add(
+  'Dual List Selector',
```

Comment:
Need to add the include props here so the storybook has info for `DualList`

Reply:
Props are now described in storybook.